packages/realm/bindgen/src/templates/wrapper.ts

```diff
@@ -0,0 +1,405 @@
+import { TemplateContext } from "@realm/bindgen/context";
```

Comment:
The removal of the platform specific wrappers and the addition of this file are the biggest part of the changes proposed here. Originally we had the thought that keeping these separate could allow for platform specific optimizations but in reality we haven't relied on that and the separation gives rise to extra unneeded complexity.